src/App.css

```diff
@@ -1,5 +1,31 @@
-#root {
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@1,600;1,700&family=Source+Sans+Pro:wght@300;400&display=swap');
+
+/* FONTS
+font-family: 'Playfair Display', serif;
+weight of 600 and 700
+
+font-family: 'Source Sans Pro', sans-serif;
+weight of 300 and 400
+*/
+
+/* COLORS
+orange: f4a431
+dark-orange: c65e26
+green: 366e41
+dark-green: 173e2e
+*/
```

Comment:
I love that y'all have started creating a kind of mini style guide here with comments. Very thoughtful! This could be good stuff to move into the README in the future as well under some kind fo "Style Guide" section or something! Did you end up using some kind of tool to pick the color scheme? Did you base it on the image you chose? Curious to know more about how you came up with the design!

Reply:
We used a Chrome extension called [ColorZilla](https://chrome.google.com/webstore/detail/colorzilla/bhlhnicpbhignbdhedgjhgdocnmhomnp?hl=en). It has a tool to let you select colors off the page. We decided to figure out the theme first, looked on unsplash for a background image that fits, and then selected four colors to be our palette (orange, green, black, and a very light gray). Using ColorZilla, we selected both a light and dark version of our green and orange from the background image.